packages/core/test/invoke.test.ts

```diff
@@ -162,7 +164,7 @@ describe('invoke', () => {
       .start();
   });
 
-  it('should forward events to services if autoForward: true', () => {
+  it('can forward events to services', () => {
```

Comment:
This test title is also kinda irrelevant now, we are not even forwarding anything here, just sending an event to a child from within a transition action. Perhaps we should rename this to something like:
```suggestion
  it('child can immediately respond to the parent with multiple events', () => {
```